[DOCS] CLI Edits (#7865)
great-expectations/great_expectations: docs/docusaurus/docs/guides/expectations/creating_custom_expectations/how_to_use_custom_expectations.md

</Prerequisites>

> There are slight differences when working with custom expectations or those contributed to GX.

Reviewer: Maybe a slight nit but when do we use `expectation` vs `Expectation`? Same note for any domain-specific concepts

> There are slight differences between working with custom Expectations and those contributed to GX.

Reviewer 2: ```suggestion
 Custom Expectations and those contributed to GX are different.
```
Why does this matter? Why should a user care?

<TabItem value="custom-expectations">

> If you're using a Custom Expectation you've built, you'll need to place it in the `great_expectations/plugins/expectations` folder of your Great Expectations deployment.

Reviewer 2: ```suggestion
If you're using a Custom Expectation you've created, add it to the `great_expectations/plugins/expectations` folder of your Great Expectations deployment.
```

When you

</TabItem>

<TabItem value="contrib-expectations">

> If you're using a Custom Expectation that is coming from the `Great Expectations Experimental` library, it will need to either be imported from there directly. To do this, we'll first need to `pip install great_expectations_experimental`.

Reviewer 2: ```suggestion
If you're using a Custom Expectation from the `Great Expectations Experimental` library, you'll need to import it.  Before you import it, run `pip install great_expectations_experimental`.
```

</TabItem>
</Tabs>

## Usage

> When you instantiate your <TechnicalTag tag="data_context" text="Data Context"/>, it will automatically make all plugins in the directory available for use,

Reviewer 2: ```suggestion
When you instantiate your <TechnicalTag tag="data_context" text="Data Context"/>, all plugins in the directory are automatically available,
```

> allowing you to import your Custom Expectation from that directory whenever and wherever it will be used.

Reviewer 2: ```suggestion
and this allows you to import your Custom Expectation from other locations.
```
